packages/core/base/core-base-browser-internal/BUILD.bazel

```diff
@@ -36,6 +36,7 @@ TYPES_DEPS = [
   "@npm//@types/react",
   "//packages/kbn-utility-types:npm_module_types",
   "//packages/kbn-config:npm_module_types",
+  "//packages/kbn-config-schema:npm_module_types",
```

Comment:
These changes highlighted a flaw in the way dependencies were automatically wired up for the previous version of this task. This might have been necessary for it to work until we tracked down a proper solution. Checking.